File: app/src/main/java/com/weiwa/ljl/weiwa/onWeiboUpdatedListener.java
package com.weiwa.ljl.weiwa;

import com.weiwa.ljl.weiwa.WeiboModel.WeiboCommentPojo;
import com.weiwa.ljl.weiwa.WeiboModel.WeiboPojo;

/**
 * Created by hzfd on 2017/1/17.
 */
public interface onWeiboUpdatedListener {
    void onUpdate(WeiboPojo pojo, int updateType);
    void onComment(WeiboCommentPojo comment);
}

File: app/src/main/java/com/weiwa/ljl/weiwa/listener/onWeiboUpdatedListener.kt
package com.weiwa.ljl.weiwa.listener

import com.weiwa.ljl.weiwa.network.WeiboCommentPojo
import com.weiwa.ljl.weiwa.network.WeiboPojo

/**
 * Created by hzfd on 2017/1/17.
 */
interface onWeiboUpdatedListener {
    fun onUpdate(pojo: WeiboPojo, updateType: Int)
    fun onComment(comment: WeiboCommentPojo)
}
